share/getters/shrex.go

```diff
@@ -132,12 +136,16 @@ func (sg *ShrexGetter) GetSharesByNamespace(
 		reqCtx, cancel := ctxWithSplitTimeout(ctx, sg.minAttemptsCount-attempt+1, sg.minRequestTimeout)
 		nd, getErr := sg.ndClient.RequestND(reqCtx, root, id, peer)
 		cancel()
-		switch getErr {
-		case nil:
+		switch {
+		case getErr == nil:
 			setStatus(peers.ResultSuccess)
 			return nd, nil
-		case context.DeadlineExceeded:
-		case p2p.ErrInvalidResponse:
+		case errors.Is(getErr, context.DeadlineExceeded),
+			errors.Is(getErr, context.Canceled):
+		case errors.Is(getErr, p2p.ErrUnavailable):
```

Comment:
Why is it called unavailable?

Reply:
We've had this discussion before - its unavailable because it means we don't have it at the moment (but may have it in a few seconds)

Reply:
I've renamed it to ErrNotFound to make it consistent to other errs in out codebase.